spec/models/user_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+describe User do
+  it { should validate_presence_of(:email)}
```

Comment:
There is a missing space before `}`

Multiply this comment to all specs in this file.